docs/integration/alerts-spikesh-integration.md

```diff
@@ -0,0 +1,50 @@
+title: Spike.sh Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Spike.sh. Use them together to get the information about each notification quickly and on any device.
+
+## In Spike.sh
+
+**1.** Create an account at [Spike.sh](https://spike.sh) if you don't have one already.
+
+To be able to receive incoming notifications we need to add and configure a new integration. To do that:
+
+**2.** Go to the **Integrations** page and click **New integration**:
+
+<img class="content-modal-image" alt="Create Spike.sh Integration - New Integration" src="../../images/integrations/create-spikesh-integration-integrations.png" title="Create Spike.sh Integration - New Integration">
+
+**3.** Chose the **Sematext** integration from the integrations list: 
+
+<img class="content-modal-image" alt="Create Spike.sh Integration - Sematext" src="../../images/integrations/create-spikesh-integration-sematext.png" title="Create Spike.sh Integration - Sematext">
+
+**4.** Provide the name, description of the integration. Chose the service that it should be added to, the escalation policy and the acknowledge time:
+
+<img class="content-modal-image" alt="Create Spike.sh Integration - Integration Description" src="../../images/integrations/create-spikesh-integration-add-integration.png" title="Create Spike.sh Integration - Integration Description">
+
+**5.** Once created click the **Copy Webhook** link:
+
+<img class="content-modal-image" alt="Create Spike.sh Integration - Copy Webhook" src="../../images/integrations/create-spikesh-integration-copy.png" title="Create Spike.sh Integration - Copy ">
+
+**6.** Finally click **Copy** button in the displayed modal window and note down the URL that was copied into the clipboard:
```

Comment:
Finally => Finally,
I assume " and note down the URL that was copied into the clipboard" is not necessary.  All one needs to do is copy the URL.